os-3.9-masteronly/scripts/node01.sh

```diff
@@ -2,8 +2,12 @@
 
 set +e
 
+# Wait until cluster will be up
 /usr/local/bin/oc get nodes
 while [ $? -ne 0 ]; do
     sleep 5
     /usr/local/bin/oc get nodes
 done
+
+# Update DHCP lease, will also update DNS servers
+dhclient
```

Comment:
That is surprising. So far the nodes always had the right  ips, node names and dns servers after a fresh vm start. dnsmasq is already ready before the first vm starts. Could you explain why this is needed?

Reply:
Oh, are the ansible-playbooks disabling dhcp?

Reply:
I do not sure why it happens, I just saw that OpenShift VM had incorrect DNS name under `/etc/resolv.conf` and it prevented to pull images from registry container.

Reply:
Ok, it concerns me a little, but if it works afterward as expected ...